templates/master/00-master/on-prem/files/keepalived-keepalived.yaml

```diff
@@ -22,17 +22,15 @@ contents:
         script "/usr/bin/timeout 1.9 /etc/keepalived/chk_ocp_script.sh"
         interval 2
         weight 20
-        rise 3
+        rise 15
         fall 3
     }
 
     vrrp_script chk_ocp_both {
```

Comment:
In the spirit of simplification, does this check really makes sense?

As I understand it, it covers a failure scenario where all haproxy go down at the same time (presumably when pushing a broken haproxy conf?) and ensures a node with a working API server gets the VIP. However, isn't that already covered by haproxy health check? In case all haproxy go down, the keepalived would all have the same priority, and the one with the highest IP gets the VIP.
 
This additional check contributes to the API flapping, as shown in the logs of this [CI job](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-api-provider-openstack/245/pull-ci-openshift-cluster-api-provider-openstack-release-4.10-e2e-openstack/1577640610280509440). Grep for `Master received advert` in the [keepalived logs](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-api-provider-openstack/245/pull-ci-openshift-cluster-api-provider-openstack-release-4.10-e2e-openstack/1577640610280509440/artifacts/e2e-openstack/gather-extra/artifacts/pods/openshift-openstack-infra_keepalived-dj5j0nmr-a0ea9-qm6t5-master-0_keepalived.log):

```
...
586:Wed Oct  5 13:35:35 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.0.78 with higher priority 60, ours 45                                                 
658:Wed Oct  5 13:37:38 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.1.29 with higher priority 65, ours 45                                                 
726:Wed Oct  5 13:38:17 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.1.29 with higher priority 65, ours 60                                                 
753:Wed Oct  5 13:38:31 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.1.29 with higher priority 65, ours 60                                                 
838:Wed Oct  5 13:40:23 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.0.78 with higher priority 65, ours 60                                                 
881:Wed Oct  5 13:41:15 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.0.78 with higher priority 65, ours 60                                                 
967:Wed Oct  5 13:43:37 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.0.78 with higher priority 60, ours 45                                                 
1099:Wed Oct  5 13:47:13 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.0.78 with higher priority 65, ours 45                                                
1213:Wed Oct  5 13:48:47 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.0.78 with higher priority 65, ours 45                                                
1288:Wed Oct  5 13:50:33 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.1.29 with higher priority 65, ours 60                                                
1391:Wed Oct  5 13:52:59 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.0.78 with higher priority 65, ours 45                                                
1517:Wed Oct  5 13:55:57 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.0.78 with higher priority 65, ours 45                                                
1590:Wed Oct  5 13:58:21 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.0.78 with higher priority 65, ours 60                                                
1681:Wed Oct  5 14:01:03 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.0.78 with higher priority 65, ours 45                                                
1811:Wed Oct  5 14:04:52 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.0.78 with higher priority 65, ours 45                                                
1977:Wed Oct  5 14:09:21 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.0.78 with higher priority 65, ours 45                                                
2096:Wed Oct  5 14:12:17 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.0.78 with higher priority 65, ours 45                                                
2167:Wed Oct  5 14:14:22 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.1.29 with higher priority 65, ours 45                                                
2318:Wed Oct  5 14:17:04 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.1.29 with higher priority 65, ours 60                                                
2355:Wed Oct  5 14:17:44 2022: (dj5j0nmr-a0ea9_API) Master received advert from 10.0.1.29 with higher priority 65, ours 60
...                                                
```